_config.yml

```diff
@@ -71,6 +71,7 @@ sidebar_categories:
     - docs/reference/analytics-components
     - docs/reference/scripts
     - docs/reference/front-commerce-prepare
+    - docs/reference/server-timings
```

Comment:
I think it would be better under the "Performance" section, as it could help developers focusing on performance optimization.